Formula/feroxbuster.rb

```diff
@@ -0,0 +1,39 @@
+class Feroxbuster < Formula
+  desc "Fast, simple, recursive content discovery tool written in Rust"
+  homepage "https://epi052.github.io/feroxbuster"
+  url "https://github.com/epi052/feroxbuster/archive/v2.3.3.tar.gz"
+  sha256 "5a244d1614fb9dc476e9be8bd1cdbb33e0b07a1fcb7285e911ae6c77a44c7327"
+  license "MIT"
+
+  depends_on "rust" => :build
+  depends_on "miniserve" => :test
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    (testpath/"wordlist").write <<~EOS
+      a.txt
+      b.txt
+    EOS
+
+    (testpath/"web").mkpath
+    (testpath/"web/a.txt").write "a"
+    (testpath/"web/b.txt").write "b"
+
+    port = free_port
+    pid = fork do
+      exec "miniserve", testpath/"web", "-i", "127.0.0.1", "--port", port.to_s
+    end
+
+    sleep 1
+
+    begin
+      exec bin/"feroxbuster", "-q", "-w", testpath/"wordlist", "-u", "http://127.0.0.1:#{port}"
+    ensure
+      Process.kill("SIGINT", pid)
+      Process.wait(pid)
+    end
```

Comment:
@carlocab asked me to make sure the server is properly shutdown, same as in `miniserve`'s tests itself.
Should I remove the `pid` related lines again then?
It's up to you guys 🤷 

This is the test code from miniserve:
```rb
test do
  port = free_port
  pid = fork do
    exec "#{bin}/miniserve", "#{bin}/miniserve", "-i", "127.0.0.1", "--port", port.to_s
  end

  sleep 2

  begin
    read = (bin/"miniserve").read
    assert_equal read, shell_output("curl localhost:#{port}")
  ensure
    Process.kill("SIGINT", pid)
    Process.wait(pid)
  end
end
```

Reply:
Yeah, this would be enough for miniserver too:
```ruby
test do
  port = free_port
  fork do
    exec "#{bin}/miniserve", "#{bin}/miniserve", "-i", "127.0.0.1", "--port", port.to_s
  end

  sleep 2

  assert_equal (bin/"miniserve").read, shell_output("curl localhost:#{port}")
end
```

Reply:
I dunno, we've had tests freeze in CI because `brew` doesn't get them all, so seems better to be explicit